pkg/frontend/openshiftcluster_putorpatch_test.go

```diff
@@ -51,7 +51,7 @@ func expectAsyncOperationDocumentCreate(asyncOperations *mock_database.MockAsync
 		)
 }
 
-func TestPutOpenShiftCluster(t *testing.T) {
+func TestPutOrPatchOpenShiftCluster(t *testing.T) {
```

Comment:
Initially I was going to make a separate test function for patch, but I don't see a lot of benefits